docs/changelogs/CHANGELOG-2.26.md

```diff
@@ -0,0 +1,320 @@
+- Update Openstack CCM to 1.30.0 ([#13498](https://github.com/kubermatic/kubermatic/pull/13498))
+- Enable Openstack config drive from seed datacenter ([#13656](https://github.com/kubermatic/kubermatic/pull/13656))
+- The OpenStack provider is now reconciling user cluster cloud resources on a regular basis ([#13191](https://github.com/kubermatic/kubermatic/pull/13191))
+
+#### VMware Cloud Director
+
+- Upgrade VCD CSI Driver to v1.6.0 ([#13706](https://github.com/kubermatic/kubermatic/pull/13706))
+    - VMware Cloud Director: CSI Driver has been upgraded to v1.6.0
+    - VMware Cloud Director: Volume expansion has been enabled in the default storage class
+
+#### VSphere
+
+- `cloud-config` handling for CCM/CSI was moved from machine-controller to KKP and cleaned up; adding `Global.ip-family` field to vSphere CSI cloud-config ([#13603](https://github.com/kubermatic/kubermatic/pull/13603))
+- Update vSphere CCM to 1.30.1 ([#13499](https://github.com/kubermatic/kubermatic/pull/13499))
+
+
+### New Features
+
+- Improve compatibility with cluster-autoscaler 1.27.1+: Pods using temporary volumes are now marked as evictable ([#13180](https://github.com/kubermatic/kubermatic/pull/13180))
+-  Add insecure/HTTP flags to the Helm sources in the ApplicationDefinitions ([#13406](https://github.com/kubermatic/kubermatic/pull/13406))  
+    - Add `insecure` and `useHTTP` options to Helm sources in `ApplicationDefinitions`. This allows to configure a plaintext or self-signed connection to an `oci://...` registry.
+    - `https://localhost` and `oci://localhost` URLs are now forbidden in `ApplicationDefinitions`. Since `localhost` would refer to the usercluster-controller-manager Pod, no such URLs should exist and the impact of this change should be non-existent
+- Add `AddonReconciledSuccessfully` condition / `Phase` to addons ([#13257](https://github.com/kubermatic/kubermatic/pull/13257))
+    - Add new `AddonReconciledSuccessfully` condition to Addon resources.
+    - Add `Phase` (New/Healthy/Unhealthy) to Addon resources (for informational purpose only, integrations should rely on the individual condition statusses) 
+- Bump Metering to 1.2.1 ([#13185](https://github.com/kubermatic/kubermatic/pull/13185))
+    - Add `format` to metering report configuration, allowing to generate JSON files instead of CSV.- Add `cloud-provider`, `datacenter` and `cluster-owner` columns to the generated metering reports
+- A new option to customize non-essential fields in the preset ([#13672](https://github.com/kubermatic/kubermatic/pull/13672))
+- Add `AllowedOperatingSystems` option for the project. This can be used to limit the allowed operating systems for a project ([#13442](https://github.com/kubermatic/kubermatic/pull/13442))
+- Add `DisplayName` for applications, this is the name displayed on the UI ([#13331](https://github.com/kubermatic/kubermatic/pull/13331))
```

Comment:
Lowercase `displayName` makes more sense here, as users will interface with the CRD instead of the Go type.